rules-tests/Php70/Rector/FuncCall/MultiDirnameRector/Fixture/skip.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Tests\Php70\Rector\FuncCall\MultiDirnameRector\Fixture;
+
+function multiDirname()
+{
+    dirname(dirname($path, $level));
+
+    dirname("foo/" . dirname($path));
+
+    dirname(dirname($path) . $foo);
+
+    foo\dirname(dirname($path));
+
+    dirname($path, 1);
+
+    dirname($path, 2);
```

Comment:
In these cases the rule would have been applied even though no actual change was made in the code